.github/workflows/e2e-crossplatform.yml

```diff
@@ -0,0 +1,119 @@
+
+name: E2E cross platform
+
+on:
+  pull_request:
+
+jobs:
+  mac:
+    runs-on: macos-latest
+    timeout-minutes: 30
+    env:
+      TEST_SYSTEM: mac
+      ELECTRON_CUSTOM_VERSION: 19.0.5
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Setup environment"
+        uses: ./.github/actions/setup-env
+        with:
+          bootstrap-packages: "@quiet/logger,@quiet/state-manager,@quiet/backend,@quiet/identity,quiet,backend-bundle,create-require-loader,e2e-tests"
+
+      - name: Download DMG
+        working-directory: ./packages/e2e-tests/Quiet
+        run: curl -LO https://github.com/TryQuiet/quiet/releases/download/quiet%400.19.17-alpha.0/Quiet-0.19.17-alpha.0.dmg
+
+      - name: Chmod
+        working-directory: ./packages/e2e-tests/Quiet
+        run: chmod +x Quiet-0.19.17-alpha.0.dmg
+
+      - name: hdiutil mount
+        working-directory: ./packages/e2e-tests/Quiet
+        run: hdiutil mount Quiet-0.19.17-alpha.0.dmg 
+
+      - name: Add to apps
+        run: cd ~ && cp -R "/Volumes/Quiet 0.19.17-alpha.0/Quiet.app" /Applications
+
+      - name: Run smoke test
+        working-directory: ./packages/e2e-tests
+        run: npm run test smoke.crossplatform.test.ts
+
+      - name: Run newUser test
```

Comment:
Is it two-players test? Maybe it's worth it include this information in the step's name?